content/updates/2017-06-12-release-notes.md

```diff
@@ -0,0 +1,29 @@
++++
+date = "2017-06-12"
+title = "Platform Release Notes: June 12, 2017"
++++
+
+Curious what’s new that you might find helpful as a cloud.gov application developer? Here are highlights from our platform updates over the past week and a half.
+<!--more-->
+
+### Changed
+
+- [Static IP addresses for communicating with external apps](https://cloud.gov/docs/apps/static-egress/#cloud-gov-egress-addresses): Outbound traffic from cloud.gov now comes from specific IP addresses to help customers open up a connection between cloud.gov and outside data centers.
+- We verified that all tenants were migrated from East/West into GovCloud! Thank you to all our customers for their patience and cooperation in making this move successful.
+- Make sure you upgrade your CloudFoundry CLI to [the latest version](https://github.com/cloudfoundry/cli/releases).
+
+### Fixed
+
+- Clarified documentation about deployer accounts: Previously actions from space-deployer service accounts displayed in logs as `devops@gsa.gov`. Now they display as the more specific and less confusing `service-account@gsa.gov`. [We've updated the documentation to reflect this](https://cloud.gov/docs/services/cloud-gov-service-account/).
```

Comment:
We can remove "Clarified documentation about deployer accounts:" here since it was both a technical change (updating the email address) and a documentation change.